src/main/resources/build.gradle.template

```diff
@@ -71,3 +71,8 @@ sourceSets {
     test.kotlin.srcDirs += "${web3j.generatedFilesBaseDir}/test/java"
 }
 
+import org.web3j.solidity.gradle.plugin.OutputComponent
+solidity {
+    outputComponents = [OutputComponent.BIN, OutputComponent.ABI, OutputComponent.METADATA]
```

Comment:
I don't like the fact that we need a special flag to make meta data upload work. I think we should revisit this re the gradle plugin. Same applies for below.

Reply:
I've pinged Xavier about this.

Reply:
What's the status of this? Was the plugin re-released with this in by default?